File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/EmailNotFoundException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class EmailNotFoundException extends NotFoundException {

    private String email;

    public EmailNotFoundException(String email) {
        super(email, ErrorCode.EMAIL_NOT_FOUND);
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/notFound/EmailNotFoundException.kt
package com.dogGetDrunk.meetjyou.common.exception.business.notFound

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class EmailNotFoundException(
    email: String,
    message: String? = null,
) : NotFoundException(ErrorCode.EMAIL_NOT_FOUND, email, message)
